packages/pluggableWidgets/notifications-native/src/Notifications.tsx

```diff
@@ -38,9 +41,14 @@ interface Notification {
 export function Notifications(props: NotificationsProps<undefined>): null {
     const listeners = useRef<Array<() => void>>([]);
     const [loadNotifications, setLoadNotifications] = useState(false);
+    const [knownIds] = useState(new Set<string>());
```

Comment:
I think you shouldn't use `useState`, you can use `useRef` instead

Reply:
Thanks Ahmad, Latest commit is using useRef